packages/world/ts/objects.ts

```diff
@@ -314,10 +326,8 @@ export interface ObjectDefinition {
   mass?: bigint;
   energy?: bigint;
   timeToGrow?: bigint;
-  isTillable?: boolean;
```

Comment:
why was isTillable removed?

Reply:
ahh I see its a meta category

Reply:
I can add it back if you need it in the client

Reply:
no need, I can just update the client to read the metaCategory